src/SparseArraysBase.jl

```diff
@@ -11,8 +11,10 @@ export SparseArrayDOK,
   oneelement,
   storedlength,
   storedpairs,
-  storedvalues
-export spzeros, sprand, sprand!
+  storedvalues,
+  spzeros,
+  sprand,
+  sprand!
```

Comment:
Could you put them alphabetically? That's the convention I've been trying to use in all of the packages.

Reply:
Thanks for changing the names, this comment is still relevant for the new names.

Reply:
Would you like alphabetically within the "topical" group, or just sorting the entire export list? Am I correct in assuming that you treat uppercase letters as coming before lowercase?

Reply:
I just sort independent of topic, since that gets too subtle and hard to maintain in my experience (better to have something you don't have to think about/use judgement calls about). And yes, I treat uppercase letters as coming before lower case, basically what you would get if you turned them into Symbols/Strings and called `sort` on them. So from that rule macros should come first but I'm not sure I've always been consistent about that.